packages/react/src/components/ContextualMenu/ContextualMenu.test.tsx

```diff
@@ -606,6 +610,8 @@ describe('ContextualMenu', () => {
     menuItems[2].focus();
     expect(document.activeElement!.textContent).toEqual('TestText 3');
     expect(document.activeElement!.className.split(' ')).toContain('is-disabled');
+
+    removeTestContainer();
```

Comment:
I don't think this approach will be reliable for cleanup if the test fails. It will need to be moved to `afterEach` somehow. (I'll look into this.)